MINIFICPP-2323 ListenTCP custom delimiter (#1753)
apache/nifi-minifi-cpp: PROCESSORS.md

### Description

> Listens for incoming TCP connections and reads data from each connection using a configurable message demarcator. For each message the processor produces a single FlowFile.

Reviewer: We should call it either delimiter or demarcator consistently.
```suggestion
Listens for incoming TCP connections and reads data from each connection using a configurable message delimiter. For each message the processor produces a single FlowFile.
```

| **Max Size of Message Queue** | 10000         |                            | Maximum number of messages allowed to be buffered before processing them when the processor is triggered. If the buffer is full, the message is ignored. If set to zero the buffer is unlimited. |
| SSL Context Service           |               |                            | The Controller Service to use in order to obtain an SSL Context. If this property is set, messages will be received over a secure connection.                                                    |
| Client Auth                   | NONE          | NONE<br/>WANT<br/>REQUIRED | The client authentication policy to use for the SSL Context. Only used if an SSL Context Service is provided.                                                                                    |

> | **Message Delimiter**         |               |                            | The delimiter is used to divide the stream into flowfiles.                                                                                                                                       |

Reviewer: The default value of '\n' is not visible, probably it's just not escaped.

> | **Consume delimiter**         | true          | true<br/>false             | If set to true then the delimiter won't be included in the resulting flowfiles.                                                                                                                  |

Reviewer: A bit more info.
```suggestion
| **Consume delimiter**         | true          | true<br/>false             | If set to true then the delimiter won't be included at the end of the resulting flowfiles.                                                                                                       |
```

> | **Consume delimiter**         | true          | true<br/>false             | If set to true then the delimiter won't be included at the end of the resulting flowfiles.                                                                                                       |

Reviewer: Most properties have capital first letters in each word.
```suggestion
| **Consume Delimiter**         | true          | true<br/>false             | If set to true then the delimiter won't be included at the end of the resulting flowfiles.                                                                                                       |
```
